main.go

```diff
@@ -77,9 +78,22 @@ type configs struct {
 	CacheLevel string `env:"cache_level,opt[none,swift_packages]"`
 }
 
-func fail(format string, v ...interface{}) {
-	log.Errorf(format, v...)
-	os.Exit(1)
+// Config ...
+type Config struct {
+	Inputs
```

Comment:
wow I never seen such syntax. What does this mean?
1) Config will have ExportMethod, UploadBitcode, etc. fields just like the Inputs (quasi-include)
2) Config will have an Input named Input typed field

?

Reply:
https://golang.org/doc/effective_go#embedding

With this type of embedding, Config will borrow the field and methods of Input.